pkg/common/key.go

```diff
@@ -0,0 +1,110 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+
+	guuid "github.com/google/uuid"
+)
+
+const (
+	KEY_DELIMETER = "/"
+	LOCKS         = "_locks"
+	COMMENTS      = "_comments"
+	INTERNAL_DB   = "_"
+)
+
+var prefix string
+
+type Key struct {
+	Prefix    string
```

Comment:
Maybe the `Prefix` should be split:
- OvsdbPrefix (such as `ovsdb`)
- ServiceName (such as `ndb`)

Reply:
why do we need to split it here? Once the variables are set the server does not need to separate them.